ui/modal/create_password_modal.go

```diff
@@ -18,24 +19,26 @@ const CreateWallet = "create_wallet_modal"
 
 type CreatePasswordModal struct {
 	*load.Load
-	randomID string
-	modal    decredmaterial.Modal
 
-	dialogTitle string
-
-	walletNameEnabled     bool
+	modal                 decredmaterial.Modal
 	walletName            decredmaterial.Editor
 	passwordEditor        decredmaterial.Editor
 	confirmPasswordEditor decredmaterial.Editor
 	passwordStrength      decredmaterial.ProgressBarStyle
 
-	isLoading      bool
-	materialLoader material.LoaderStyle
+	isLoading         bool
+	isMinimizable     bool
```

Comment:
The modal is actually being dismissed not minimized. 
```suggestion
	isCancelable     bool
```